api/v1/controllers/contests/index.js

```diff
@@ -0,0 +1,86 @@
+const { query, body, validationResult } = require("express-validator");
+const utils = require("./utils");
+const logic = require("./logic");
+const constants = require("./constants");
+
+// GET /api/v1/contests?offset={offset}&limit={limit}
+async function getAllContests(req, res, next) {
+  try {
+    const contests = await logic.getAllContests(
+      parseInt(req.query.offset, 10),
```

Comment:
10 là gì vậy Kiên?
Mình có nên define trong util 1 hàm parse number k? Để mọi người parse 1 kiểu thôi.
StrToInt(): return parseInt(x, 10)

Reply:
10 là hệ THẬP phân
không nên tạo hàm strToInt, bởi vì hàm parseInt phổ biến và dễ hiểu

Reply:
https://www.w3schools.com/jsref/jsref_parseint.asp

> Syntax
> parseInt(string, radix)

Parameter | Description
-- | --
string | Required. The string to be parsed
radix | Optional. A number (from 2 to 36) that represents the numeral system to be used

Reply:
Okie